repondeur/db_migrations/versions/b86ab6e391df_add_initial_tables.py

```diff
@@ -0,0 +1,77 @@
+"""Add initial tables
```

Comment:
Which was the command to generate that file? `alembic revision --autogenerate -m "Add initial tables"`?

Reply:
Yes (just add the config file param):
```
alembic -c development.ini revision --autogenerate -m "Add initial tables"
```

Reply:
Then clean up comments and run black...

Reply:
Added some notes in the PR description with the process to generate the initial boilerplate.